test/clj/game/cards/programs_test.clj

```diff
@@ -3500,6 +3500,36 @@
               (card-ability state :runner houdini 1))
             "Runner has not been charged, strength hasn't changed"))))
 
+(deftest hush
+  (do-game
+   (new-game {:corp {:deck [(qty "Hedge Fund" 2)]
+                     :hand ["Tollbooth"]
+                     :credits 10}
+              :runner {:deck [(qty "Sure Gamble" 2)]
+                       :hand ["Hush"]
+                       :credits 20}})
+   (play-from-hand state :corp "Tollbooth" "R&D")
+   (take-credits state :corp)
+   (take-credits state :corp)
+   (core/gain state :runner :click 1)
+   (let [toll (get-ice state :rd 0)
+         credits (:credit (get-runner))]
+     (run-on state "R&D")
+     (rez state :corp toll)
+     (run-continue state)
+
```

Comment:
I know it can help with readability but I prefer to have no empty spaces in tests.

Reply:
Good to know, thanks!